orbit/cmd/orbit/orbit.go

```diff
@@ -903,37 +905,44 @@ func (d *desktopRunner) interrupt(err error) {
 	}
 }
 
-// shell out to osquery (on Linux and macOS) or to wmic (on Windows), and get the system uuid
-func getUUID(osqueryPath string) (string, error) {
+// shell out to osquery (on Linux and macOS) or to wmic (on Windows), and get
+// the system uuid and serial number
+func getUUIDAndSerial(osqueryPath string) (uuid, serial string, err error) {
 	if runtime.GOOS == "windows" {
+		// NOTE: Windows uses a different approach as there were issues at the time
+		// with shelling out to osquery - from what the team remembers it would
+		// sometimes fail due to the osquery process not being ready yet. A recent
+		// CI run without the Windows special-case did succeed, but since we don't
+		// need the serial number for Windows at the moment, we opted to keep the
+		// code as it is.
 		uuidData, uuidSource, err := platform.GetSMBiosUUID()
 		if err != nil {
-			return "", err
+			return "", "", err
 		}
 
 		log.Debug().Msgf("UUID source was %s.", uuidSource)
-
-		return uuidData, nil
+		return uuidData, "", nil
 	}
-	type UuidOutput struct {
-		UuidString string `json:"uuid"`
+	type Output struct {
+		UuidString     string `json:"uuid"`
+		HardwareSerial string `json:"hardware_serial"`
 	}
 
 	args := []string{"-S", "--json", "select uuid from system_info"}
```

Comment:
```suggestion
	args := []string{"-S", "--json", "select uuid, hardware_serial from system_info"}
```

Reply:
Oh man, that's embarrassing... :D Great catch!